graftm/hmmer.py

```diff
@@ -675,88 +693,83 @@ def d_search(self, files, args, run_stats, base, unpack, raw_reads, euk_check):
             The reads to be searched.
         euk_check : bool
             True False, whether to check the entire sample for euk reads.
```

Comment:
"True indicates the sample will be checked for eukaryotic reads, False indicates not."